project/Build.scala

```diff
@@ -349,6 +351,12 @@ object DottyInjectedPlugin extends AutoPlugin {
      )
    )
 
+   lazy val `dotty-core` = project
```

Comment:
That is the intention of `dotty-core`, once @sjrd is done with the restructuring this could simply be renamed to `dotty`.